docs/network-setup.md

```diff
@@ -0,0 +1,71 @@
+# Getting Started Firecracker Network Setup
+
+This is a very simple quick-start guide to getting a Firecracker guest connected to the network. If you're using Firecracker in production, or even want to run multiple guests, you'll need to adapt this setup.
+
+## On The Host
+
+The first step on the host is to create a `tap` device:
+
+```bash
+sudo ip tuntap add tap0 mode tap
+```
+
+Then you have a few options for routing traffic out of the tap device, through your host's network interface. One option is NAT, set up like this:
+
+```bash
+DEV=eth0
+sudo ip addr add 169.254.0.1/24 dev tap0
```

Comment:
I'd also replace all occurrences of `tap0` with a variable; the same goes for the IP address. I think each user knows their network setup best.